sly-asdf-flymake.el

```diff
@@ -304,15 +301,12 @@ not pass the diagnostic's buffer to `make-overlay`."
           (popup-tip (sly-asdf-flymake-base-diagnostic-text (car diags)) :point point))))))
 
 
+;; Activate (upon value of `sly-asdf-enable-experimental-syntax-checking')
+(add-hook 'sly-connected-hook
+          ;; MG: Investigate race, due to when ASDF loads?
+          ;; OC: Is this still a problem after decoupling from sly-asdf?
```

Comment:
Good point, will need to to take another look and see if I can still repro this.